src/core/meshcop/secure_transport.cpp

```diff
@@ -1022,25 +1022,31 @@ void SecureTransport::HandleMbedtlsDebug(int aLevel, const char *aFile, int aLin
 
     switch (aLevel)
     {
+    case 0:
```

Comment:
How did you come up with the new mapping of mbedtls levels?

I found the following:
https://github.com/openthread/openthread/blob/6c661ee55a3229ce8ece0d8c015be5237ca16534/third_party/mbedtls/repo/include/mbedtls/debug.h#L135-L150

- This PR is nownow mapping `0` to `kLogLevelCrit` but it seems to mean "no debug"
- Also it is adding `case 5` which is not mentioned.

Reply:
Based on the mbedtls documentation, I would suggest the following mapping:
```
- Debug levels 
  - 0 No debug      -> LOG_LEVEL_NONE
  - 1 Error         -> LOG_LEVEL_CRIT
  - 2 State change  -> LOG_LEVEL_NOTE
  - 3 Informational -> LOG_LEVEL_INFO
  - 4 Verbose       -> LOG_LEVEL_DEBG
  ```